samples/soundtest.lobster

```diff
@@ -0,0 +1,173 @@
+// Extension of shooter tutorial tut7 here with background music, enemy sounds and "hot" pause
+
+import std
+import vec
+import color
+
+fatal(gl_window("Shooter Tutorial", 640, 480))
+
+check(gl_set_font_name("data/fonts/US101/US101.ttf") and gl_set_font_size(32), "can\'t load font!")
+
+// load music and sound files
+let bgmusic = play_wav("data/sounds/Cyber-Dream-Loop.wav", -1) // by Eric Matyas from https://soundimage.org/looping-music/
```

Comment:
6MB still too big.. if there's no music that is small, lets just add an environment sound loop?

Reply:
Ok. I hope one of us or someone will add libvorbis to the external sources and allow .ogg files (easily > 10x smaller)